Rideau.xcodeproj/project.pbxproj

```diff
@@ -636,7 +636,7 @@
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
-				INFOPLIST_FILE = "$(SRCROOT)/RideauDemo/Info.plist";
+				INFOPLIST_FILE = "$(SRCROOT)/Rideau/Info.plist";
```

Comment:
This breaks the Demo app, and it doesn't change the fact that your app has imported files from the rest of the Demo project.

Please use Cocoapods or Carthage to import Rideau as those are the only currently supported distribution (see https://github.com/muukii/Rideau#installation)

Reply:
Hi John, have you tried to run the demo app? I use carthage to import it. Everything works now after the change i made. The demo project is untouched. The framework did use the info.plist file from the demo and that is what i fixed.

Reply:
Thanks for the clarification, I might have misunderstood where this setting was changed. I'll check the installations.